modules/ROOT/pages/asm-secret-group-creation-task.adoc

```diff
@@ -72,67 +54,99 @@ If you are uploading a JKS, PKCS12, or JCEKS truststore file, you must also prov
 . Click *Save*.
 
 == Add a Certificate Pinset
 
 Add a concatenated list of PEM certificates to the secret group. 
 
-. In the *Secret Groups* list view, select the secret group to which to add a certificate pinset. 
+. In the *Secret Groups* list view, select the secret group to which to add a certificate pinset, and click *Edit*. 
 . Select *Certificate Pinset* in the menu on the left, and click *Add Certificate Pinset*. 
 . In the *Certificate Pinset* screen, add the required information. +
-* *Name* - Enter a name for the certificate pinset. 
-* *Certificate File* - Click *Choose File* and select the PEM formatted CA certificate to upload.
-* *Expiration Date* (Optional) - Select the expiration date for the certificate.
+* *Name* +
+Enter a name for the certificate pinset. 
+* *Certificate File* +
+Click *Choose File* and select the PEM formatted CA certificate to upload.
+* *Expiration Date* +
+Select the expiration date for the certificate.
+. Click *Save*.
 
 == Add a Shared Secret
 
 Add a shared secret users can use for authentication. 
 
-. In the *Secret Groups* list view, select the secret group to which to add a shared secret. 
+. In the *Secret Groups* list view, select the secret group to which to add a shared secret, and click *Edit*. 
 . Select *Shared Secret* in the menu on the left, and click *Add Shared Secret*. 
 . In the *Add Shared Secret* screen, add the required information. 
 +
-* *Name* - Enter a name for your shared secret.
-* *Type* - Select the shared secret type from the drop-down menu.
- ** *Username Password* - Provide a username and password.
- ** *Symmetric Key* - Provide a Base64 string containing symmetric key.
- ** *S3 Credential* - Provide the access key ID and the secret access key to an S3 bucket.
- ** *Blob* - Provide a base64-encoded value.
+* *Name* +
+Enter a name for your shared secret.
+* *Type* +
+Select the shared secret type from the drop-down menu.
+ ** *Username Password* +
+ Provide a username and password.
+ ** *Symmetric Key* +
+ Provide a Base64 string containing symmetric key.
+ ** *S3 Credential* +
+ Provide the access key ID and the secret access key to an S3 bucket.
+ ** *Blob* +
+ Provide a base64-encoded value.
 . Click *Save*. 
 
+//// 
 == Add a List of Revoked Certificates 
 
+A Certificate Revocation List (CRL) distributor is an entity that creates and maintains a list of certificates issued by the CA that are no longer trusted because their associated private keys, or a signing CA, were compromised.
+
+Anypoint Secrets Manager lets you create CRL distribution points to query a CRL Distributor for a specific CA certificate. +
+To configure a CRL distributor, you must provide a URL of the CRL distributor, a CA certificate, and a frequency to check the list for updates. +
+As some CRL lists can grow very large, you can provide an optional Delta CRL Issuer URL (if the CRL distributor supports it) to retrieve deltas to the CRL list, instead of retrieving the entire list.
```

Comment:
```suggestion
Because some CRL lists can grow very large, you can provide an optional Delta CRL issuer URL (if the CRL distributor supports it) to retrieve deltas to the CRL list, instead of retrieving the entire list.
```